stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -421,18 +421,17 @@ SparseVector{Tv}(s::AbstractVector{Tv}) where {Tv} = SparseVector{Tv,Int}(s)
 
 SparseVector(s::AbstractVector{Tv}) where {Tv} = SparseVector{Tv,Int}(s)
 
+# copy-constructors
+SparseVector(s::SparseVector{Tv,Ti}) where {Tv,Ti} = SparseVector{Tv,Ti}(s)
+SparseVector{Tv}(s::SparseVector{<:Any,Ti}) where {Tv,Ti} = SparseVector{Tv,Ti}(s)
+function SparseVector{Tv,Ti}(s::SparseVector) where {Tv,Ti}
+    copyind = Vector{Ti}(nonzeroinds(s))
+    copynz = Vector{Tv}(nonzeros(s))
+    SparseVector{Tv,Ti}(length(s::SparseVector), copyind, copynz)
```

Comment:
I know this was here before but could remove the `::SparseVector` here?

Reply:
you mean in the `length...` ?

Reply:
Yes

Reply:
done